### DISCUSSION TITLE
How to insert image into table cell?

### DISCUSSION THREAD

**Discussion by JOY** [GitbookIO/community#447] [1. How to]
Hi,

I'd like to insert and image into a table cell, how can I do that?
Example (not Gitbook): https://docs.onbeam.com/chain/builder/contract-addresses

Thanks.

**Comment by JaniceManwiller** [comment]
You can add an inline image (press / and then select **Inline Image**), but note that inline images are limited in size, since I think they are intended to do something like add an icon in the middle of a sentence.

I'm also extremely frustrated at the limits of what can be added to a table cell. Would want to press Enter to add a new paragraph and also be able to add short lists.
